### PR TITLE
Update documentation on Podman version and quickstart

### PR DESCRIPTION
**Pull request by windsource**
Issues: #1

# Definition of Done

The PR shall be merged only if all items mentioned in 
[CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute)
have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: doc/mkdocs.yml [hunk 1]
```diff
@@ -55,6 +55,7 @@ markdown_extensions:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
+  - pymdownx.details
```
Comment (krucod3) [inline doc/mkdocs.yml:58]: Why do we need this extension?

Comment (krucod3) [reply doc/mkdocs.yml:58]: Everything works also without it.

File: tools/generate_docs.sh [hunk 2]
```diff
@@ -27,9 +27,9 @@ protoc --plugin=protoc-gen-doc="/usr/local/bin/protoc-gen-doc" --doc_out="$targe
 echo "Generate Markdown from ./api/proto/ankaios.proto done."
 
 if [[ "$1" = serve ]]; then
-    mkdocs serve -f "$target_dir/mkdocs.yml"
+    mkdocs serve --config-file "$target_dir/mkdocs.yml"
```
Comment (krucod3) [inline tools/generate_docs.sh:30]: I tried to serve the documentation locally to check how it works and I'm getting warnings:
```
WARNING -  [10:06:55] "GET /versions.json HTTP/1.1" code 404
```
I searched a bit and found this issue here: https://github.com/squidfunk/mkdocs-material/issues/3164

Should we use `mike` here?

```suggestion
    mike serve --config-file "$target_dir/mkdocs.yml"
```

**Review by krucod3 — COMMENTED** [review]
Looks good. Only one suggestion.

File: doc/mkdocs.yml (hunk 1 above)
Comment (krucod3) [reply doc/mkdocs.yml:58]:
```suggestion
```

File: tools/generate_docs.sh (hunk 2 above)
Comment (windsource) [reply tools/generate_docs.sh:30]: It's not that easy. `mkdocs serve` and `mike serve` are quite different:

* `mkdocs serve`: Build the documentation and watch for changes and serve the result.
* `mike serve`: Uses an already built documentation which has been committed to the `gh-pages` branch. So it is neither building nor watching.

From my point of view the warning is not that urgent and could be ignored. For preview we do not need the features of mike like switching between different versions.

Comment (krucod3) [reply tools/generate_docs.sh:30]: Alright.

**Review by krucod3 — APPROVED** [review]
LGMT 👍

**Pull request merged**